### ISSUE TITLE
Missing storjlabs/storagenode:latest docker image for arm/v7

### ISSUE THREAD

**Issue by hostingnuggets** [storj/storj#4716]
Hello

I can't update the storjnode Docker image running on my Raspberry Pi 4 with Raspbian 10 anymore due to a missing image for this architecture as you can see below:

```
$ docker pull storjlabs/storagenode:latest
latest: Pulling from storjlabs/storagenode
no matching manifest for linux/arm/v7 in the manifest list entries
```

Could you please fix this? It was working a few days ago...

For reference here is the exact details of the Linux kernel running on that hardware:

```
$ uname -a
Linux node 5.10.63-v7l+ #1496 SMP Wed Dec 1 15:58:56 GMT 2021 armv7l GNU/Linux
```

Thank you

**Comment by Erikvv** [comment]
There is a issue with the networking stack for ARM 32-bit.

https://github.com/storj/storj/commit/e486585853f779b9f4d287b4254f040324ea81a3 wil fix it, but this commit isn't in a release tag yet (the github GUI will show that). cc @profclems

**Comment by profclems** [comment]
The network issue on alpine is fixed by https://github.com/storj/storj/commit/c641f4c9aca8ec214e738d59c0bc74dfac85b6e9 instead which is released as part of [v1.52.2](https://github.com/storj/storj/releases/v1.52.2) 

We decided not to tag the arm32 images until we are sure we won’t break any nodes in production. We advice that you pull the image specified in this [changelog](https://forum.storj.io/t/changelog-v1-52-2/18025?u=clement) and test with your testnodeto verify that it works for your node: https://forum.storj.io/t/changelog-v1-52-2/18025?u=clement

**Comment by hostingnuggets** [comment]
I don't have the luxury of having a test node so I will have to wait that the arm32 image gets tagged correctly again. @profclems  any time estimates when this will be the case?
